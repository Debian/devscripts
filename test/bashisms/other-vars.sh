#!/bin/sh

test() {
    echo $FUNCNAME BASHISM
}

test

echo $DIRSTACK BASHISM
echo $SECONDS BASHISM
echo $TMOUT BASHISM
echo $TIMEFORMAT BASHISM
TMOUT=2 # BASHISM
read REPLY
TIMEFORMAT='' # BASHISM
