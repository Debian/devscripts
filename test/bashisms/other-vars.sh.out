possible bashism in bashisms/other-vars.sh line 4 ($FUNCNAME):
    echo $FUNCNAME BASHISM
possible bashism in bashisms/other-vars.sh line 9 ($DIRSTACK):
echo $DIRSTACK BASHISM
possible bashism in bashisms/other-vars.sh line 10 ($SECONDS):
echo $SECONDS BASHISM
possible bashism in bashisms/other-vars.sh line 11 ($TMOUT):
echo $TMOUT BASHISM
possible bashism in bashisms/other-vars.sh line 12 ($TIMEFORMAT):
echo $TIMEFORMAT BASHISM
possible bashism in bashisms/other-vars.sh line 13 (TMOUT=):
TMOUT=2 # BASHISM
possible bashism in bashisms/other-vars.sh line 15 (TIMEFORMAT=):
TIMEFORMAT='' # BASHISM
